src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -86,7 +86,7 @@ class PrisonerIepLevelReviewService(
         locationId = iepMigration.locationId,
         sequence = 0,
         current = iepMigration.current,
-        reviewedBy = iepMigration.userId,
+        reviewedBy = iepMigration.userId ?: "NOMIS",
```

Comment:
I don't know how I feel about this 🤔 

I see the value of avoiding a `null` value, but the issue with this is that field values are for user IDs, so the client may make a request to get user information (e.g. to show the user full name or something) - and in this case it would not work.

(Unless there is a special user with ID `"NOMIS"` used across DPS for these kind of situations I'm not aware of)

The clients may have special logic to handle this special user ID but it somehow doesn't feel right 🤔

Reply:
i would have thought we'd keep it `null` (which is true to NOMIS) and simply choose to present that in some way in DPS. "NOMIS" would not necessarily be meaningful to users of DPS and i'm sure the team would want involvement in choosing the right word

Reply:
Very good points - have changed `prisoner_iep_level.reviewedBy` to be nullable and removed setting it to `NOMIS` during the migration flow